.github/workflows/cibuildwheel.yml

```diff
@@ -70,7 +70,6 @@ jobs:
           && sh .ci/build_deps.sh
         CIBW_BEFORE_BUILD_MACOS: brew install hdf5 netcdf
         CIBW_TEST_SKIP: "*_arm64"
-        CIBW_TEST_REQUIRES: pytest cython
```

Comment:
Looks like cibuildwheel still needs this regardless of it being declared in the pyporoject.toml. The levels of abstraction and isolation when building wheels is quite confusing.